cmd/chisel/cmd_cut_test.go

```diff
@@ -317,6 +315,47 @@ var cutTests = []cutTest{{
 		"/other-dir/":     "dir 0755",
 		"/other-dir/file": "symlink ../dir/file",
 	},
+}, {
+	summary: "Copyright is automatically extracted if exists",
```

Comment:
What we want to see if the manifest, we are already testing this in `slicer_test.go`. Can you please add it?

In fact I would suggest adding the copyright to an existing test because this is duplicating the specific integration test we have in slicer_test.go.